explore/src/main/scala/explore/observationtree/ViewCommon.scala

```diff
@@ -10,24 +10,33 @@ import explore.components.ui.ExploreStyles
 import explore.model.Focused
 import explore.model.Focused._
 import explore.model.ObsSummary
+import japgolly.scalajs.react.Callback
 import japgolly.scalajs.react.ReactEvent
 import japgolly.scalajs.react.vdom.TagMod
 import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.model.Observation
 import react.beautifuldnd._
 
 trait ViewCommon {
   def focused: View[Option[Focused]]
 
-  def renderObsBadge(obs: ObsSummary, highlightSelected: Boolean = true): TagMod =
+  def renderObsBadge(
+    obs:               ObsSummary,
+    highlightSelected: Boolean = true,
+    forceHighlight:    Boolean = false // if true, overrides highlightSelected
```

Comment:
Why do we need `forceHighlight`?

Reply:
Ah, is it to highlight all observations in a group?

Reply:
Ah, is it to highlight all observations in a group?

Reply:
Exactly.